frontend/packages/kubevirt-plugin/src/components/info-messages/vmi-details-page-info-message.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { HintBlock, ExternalLink } from '@console/internal/components/utils';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { VirtualMachineInstanceModel, VirtualMachineModel } from '../../models/index';
+import { K8sResourceCommon } from '@console/internal/module/k8s';
+
+const InfoMessageHintBlock: React.FC<InfoMessageHintBlockProps> = ({ name, namespace }) => {
+  const { t } = useTranslation();
+  const [resource] = useK8sWatchResource<K8sResourceCommon>({
+    kind: VirtualMachineInstanceModel.kind,
+    name,
+    namespace,
+  });
+  const isOwnedByVM = !!resource?.metadata?.ownerReferences.find(
+    ({ kind }) => kind === VirtualMachineModel.kind,
+  );
+
+  return (
+    resource &&
+    !isOwnedByVM && (
+      <HintBlock
+        className="kubevirt-vmi-details-page-info-message__hint-block"
+        title={t('kubevirt-plugin~`Virtual Machine Instance {{name}}`', { name })}
+      >
+        <p>
+          {t(
+            'kubevirt-plugin~Consider using a Virtual Machine that will provide additional management capabilities to a VirtualMachineInstance inside the cluster.',
+          )}
+        </p>
+        <ExternalLink
+          href="https://kubevirt.io/user-guide/#/architecture?id=virtualmachine"
+          text={t('kubevirt-plugin~Learn more')}
+        />
+      </HintBlock>
+    )
+  );
+};
+
+type InfoMessageHintBlockProps = {
+  name: string;
+  namespace: string;
+};
+
+export default InfoMessageHintBlock;
```

Comment:
hmm, `export default` is not polite, some editors will ignore it when suggesting auto imports, and users may misspell the component when importing, can we use named import, WDYT ?

Reply:
Actually its a convention in console to use camel case for component files (VMIDetailsPageInfoMessage.tsx) and use default export. Its just kubevirt plugin that didnt align yet :) I would keep the default export as is

Reply:
Im going with convention, i agree with @rawagner.
 not related to this pr but same goes for nested components btw which is heavily used in kubevirt :)

Reply:
+1 from me to use console convention :+1: